programs/testapp.c

```diff
@@ -316,8 +316,8 @@ static char *get_module(const char *module) {
     static char buffer[1024];
 
     assert(getcwd(buffer, sizeof(buffer)));
-    strcat(buffer, "/.libs/");
-    strcat(buffer, module);
+    strncat(buffer, "/.libs/", sizeof(buffer) - strlen(buffer) - 1);
```

Comment:
Looks like buffer is empty and no need to subtract strlen()

Reply:
You're right, but it's safer this way as we don't make any expectations about the buffer's state 😄